flytekit/image_spec/image_spec.py

```diff
@@ -30,56 +30,57 @@ class ImageSpec:
     """
     This class is used to specify the docker image that will be used to run the task.
 
-    Args:
-        name: name of the image.
-        python_version: python version of the image. Use default python in the base image if None.
-        builder: Type of plugin to build the image. Use envd by default.
-        source_root: source root of the image.
-        env: environment variables of the image.
-        registry: registry of the image.
-        packages: list of python packages to install.
-        conda_packages: list of conda packages to install.
-        conda_channels: list of conda channels.
-        requirements: path to the requirements.txt file.
-        apt_packages: list of apt packages to install.
-        cuda: version of cuda to install.
-        cudnn: version of cudnn to install.
-        base_image: base image of the image.
-        platform: Specify the target platforms for the build output (for example, windows/amd64 or linux/amd64,darwin/arm64
-        pip_index: Specify the custom pip index url
-        pip_extra_index_url: Specify one or more pip index urls as a list
-        pip_secret_mounts: Specify a list of tuples to mount secret for pip install. Each tuple should contain the path to
+    Attributes:
```

Comment:
Updated all this Doc to fix linter

Reply:
cc @ppiegaze - your upcoming flytekit PR renders these changes moot right?  Along with the monodocs build?  I've been out of the loop on the doc building stuff for a while, do you know what the plan for that is?  or maybe @cosmicBboy?

Reply:
For context, I was doing my best here to get tests passing under the assumption that I was upholding the current standards. I am not aware of ongoing changes and potential CICD inconsistencies.